code/game/objects/structures/crates_lockers/closets/secure/genpop.dm

```diff
@@ -0,0 +1,97 @@
+/obj/structure/closet/secure_closet/genpop

```

Comment:
Would be really nice if we got a sprite for the lockers that had something making them more distinctive.